makefile

```
@@ -1,25 +1,48 @@
 # Default target
 .PHONY: all
-all: openttd-sans openttd-serif openttd-small openttd-mono
+all: ttf openttd-sans openttd-serif openttd-small openttd-mono
 
+# Output directory
+ttf:
+	mkdir ttf
```

```suggestion
	mkdir -p ttf
```

Recommend `-p` so it doesn't fail if make decides that the ttf directory is out of date (which can happen)

---

Done, and updated making preview directory too.